specs/Storage.Pickers/FileSavePicker.md

```diff
@@ -85,7 +96,7 @@ savePicker.SuggestedFileName(L"NewDocument");
 
 // (Optional) Sets the folder that the file save dialog displays when it opens.
 //     If not specified or the specified path doesn't exist, defaults to the last folder the user visited.
-savePicker.SuggestedFolder = L"C:\\MyFiles",
+savePicker.SuggestedFolder(L"C:\\MyFiles");
```

Comment:
What is the difference between FileSavePicker.SuggestedStartFolder and FileSavePicker.SuggestedFolder? They seem to do the same thing.

Should we simply continue using FileSavePicker.SuggestedFolder (and not add FileSavePicker.SuggestedStartFolder), and add FileOpenPicker.SuggestedFolder + FolderPicker.SuggestedFolder?